packages/react-components/react-charts-preview/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -529,7 +535,7 @@ export const CartesianChart: React.FunctionComponent<IModifiedCartesianChartProp
         defaultTabbableElement={`#${idForDefaultTabbableElement}`}
         {...svgFocusZoneProps}
       > */}
-      <div className={classes.chartWrapper}>
```

Comment:
remove the commented out code for focus zone

Reply:
Okay